t/copy_input_fastq.t

```diff
@@ -0,0 +1,27 @@
+#!/bin/usr/env perl
+use warnings;
+use strict;
+
+use lib '/Warehouse/Users/csieffert/core-phylogenomics/lib';
```

Comment:
Could you switch from using the absolute path here to using a relative path.  This can be accomplished by getting the path to `copy_input_fastq.t` using the [FindBin](http://perldoc.perl.org/FindBin.html) module in Perl, and adding onto this value.  For example:

```perl
use FindBin;
use lib $FindBin::Bin.'/../lib' # equivalent to 'use lib /path/to/t/../lib'
```

See, for example, https://github.com/apetkau/core-phylogenomics/blob/59bb1ee610cfef9f04553e3dc861ffb31a0c6535/t/pipeline_blast.t#L10